README.md

```diff
@@ -91,6 +91,16 @@ To create the .json file:
 - create a service account. Only service account can have application keys.
 - add service account to bucket and give it permissions to read/write/list as appropriate
 
+If you are unable to use a service account, but have logged into a specific Google account
+that has access to a certain bucket, you can use `--credential_helper` option instead.
+
+To use this feature, copy the `cred_helper_template.py` file to `cred_helper.py` and replace
+the `USER` variable with the username you would like to use. This user needs to be logged in
+and have access to the bucket specified. An example `.bazelrc` file for this scenario would
+look like this:
+  
+    build --credential_helper=cred_helper.py --remote_cache=https://storage.googleapis.com/megaboom-bazel-artifacts --remote_cache_compression=true
```

Comment:
absolute path required for cred_helper.py

Reply:
Done